Dfe.Academies.Academisation.IService/ServiceModels/Legacy/Extensions/ProjectNoteExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Dfe.Academies.Academisation.Domain.Core.ProjectAggregate;
+using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate;
+
+namespace Dfe.Academies.Academisation.IService.ServiceModels.Legacy.Extensions
```

Comment:
Should this live in the interface project?

Reply:
Good point - it was moved (but seemingly copied) so this is not used. Shall remove this one.